crds/tenant-crd.yaml

```diff
@@ -1,12 +1,32 @@
-apiVersion: apiextensions.k8s.io/v1
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Why we use v1beta1 for CRDs? It has been deprecated since Kubernetes v1.16.

[see here](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning/#specify-multiple-versions)

```
# Deprecated in v1.16 in favor of apiextensions.k8s.io/v1
apiVersion: apiextensions.k8s.io/v1beta1
kind: CustomResourceDefinition
```

Reply:
Because support for newer was added to controller-gen 28 days ago. 
https://github.com/kubernetes-sigs/controller-tools/releases/tag/v0.4.0

Do you think we need to update to it?

Reply:
Think yes, unless huge impacts on the controller, imho.

Reply:
Can you please create an issue on me, i will attach PR to close it tomorrow?

Reply:
@bsctl please refer to https://github.com/clastix/capsule/issues/90#issuecomment-694134836
we had to use older versions until controller-runtime libs are updated

Reply:
@MaxFedotov please go with this PR. Thanks